docs/source/release_notes.rst

```diff
@@ -15,6 +15,7 @@ Release Notes
         * Added custom-index support for `reset-index-get_prediction_vs_actual_over_time_data` :pr:`1494`
         * Fix ``generate_pipeline_code`` to account for boolean and None differences between Python and JSON :pr:`1524`
         * Set max value for plotly and xgboost versions while we debug CI failures with newer versions :pr:`1532`
+        * Set only XGBoost version :pr:`1533`
```

Comment:
@bchen1116 I suggest you update this to something like "Undo pinning plotly version", since xgboost isn't related to this, it just happened to end up in the update-deps PR at the same time :)